plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPath.java

```diff
@@ -148,11 +131,11 @@ public static Library collectLibraryFiles(IJavaProject javaProject, List<Library
         }
       }
     } catch (IOException | CoreException ex) {
-      logger.log(Level.WARNING, "Error loading previous libraries", ex);
+      logger.log(Level.WARNING, "Error loading previous libraries", ex); //$NON-NLS-1$
     }
     
     Library masterLibrary = new Library(CloudLibraries.MASTER_CONTAINER_ID);
-    masterLibrary.setName("Google APIs"); //$NON-NLS-1$
+    masterLibrary.setName(Messages.getString("google.api.libraries")); //$NON-NLS-1$
```

Comment:
This is the container name we see in the package explorer. There seems little value of translating this name "Google APIs"?

Reply:
The ordering may change in a RTL language.

Reply:
This should probably be renamed to "Google Cloud Platform Libraries" then